### DISCUSSION TITLE
Palauan Children’s Book on Amazon

### DISCUSSION THREAD

**Discussion by johnbent** [tekinged/missing#433] [Q&A]
Hello, I just discovered this book on Amazon:

https://www.amazon.com/First-Palauan-Picture-Word-Book/dp/1542767652/ref=sr_1_18?crid=23PGKTUL6VEJT&keywords=gerard+aflague+books&qid=1687912008&sprefix=gerard+aflague+books%2Caps%2C93&sr=8-18#customerReviews

First Palauan Picture Word Book

![image](https://github.com/tekinged/missing/assets/2047294/78813d37-f033-4c92-bd9b-31f3bfa67ce5)

@ChrisPerrette , you reviewed it and said it's good but the other two reviews said it's very inaccurate. But your review was many years old. Now that you've learned more Palauan, do you still think it's great? I'm thinking about getting it but don't want to if it's garbage like those other two reviews suggest.

**Comment by Lukes-Imeyuns** [comment]
Wow! Thanks for sharing. I just ordered it! Lol.

**Comment by s-o-u-a-n-g** [comment]
I've seen that for sale and always thought it was weird since it wasn't written by a Palauan or someone who has lived in Palau. 

I wish my mom's books "Fishing in Melekeok" and "Palauan A, B, Ch" were still in print. They show up on eBay and other places from time to time. She self published and only Surangel agreed to carry them.

Joleen Ngoriakl also published a book, but I also don't know where it is sold or if it's still in print. I think the title had "Sechelik" in it.

**Comment by Lukes-Imeyuns** [comment]
Thank you, Souang. I'll keep an eye out for those books. :)

So I ordered a copy of this book. After the first couple of pages, I
thought, ok, not too bad. But after a couple more I was disappointed. But
only because he could have used other objects and things that could
actually be translated to Palauan.

Here are some words that I thought were kind of silly...

Karpet
Nektai
Daimond
Kreips
Skorpion - And it's iusbeluu
Helikapter
Tsietski
Uakon - We use riakang actually.
Babier el taor - We just say paper towel. Lol
Blil a ongdelel a udoud - bank
Kingall ra chelsel a blai - couch

In the book he says one of his favorite trips was to Palau.

I attached a picture with his email he shares for ideas for Palauan books.

He lives in Colorado. I ought to have coffee with this guy... 😅
